airflow/models/dag.py

```diff
@@ -290,6 +290,9 @@ def __init__(
         self._access_control = access_control
         self.is_paused_upon_creation = is_paused_upon_creation
 
+        # A stringified DAG can only be used in the scope of scheduler and webserver.
```

Comment:
Can you add more comments around why it can only be used in scheduler and webserver?

Reply:
Added a class SerializedDAG with more comments to explain this.